algolia-crawler.js

```diff
@@ -39,6 +57,11 @@ new Crawler({
             lvl2: 'article h2',
             lvl3: ['article h3', 'article h4'], // an array is a list of fallbacks. Authors sometimes omit h3 to skip the side nav.
             lvl4: ['article h4', 'article h5'],
+            products: {
```

Comment:
this is the one I'm talking about in my previous comment .. do we still need it ? since now we have `_tags`